src/components/scenes/FioAddressDetailsScene.js

```diff
@@ -106,14 +127,22 @@ const getStyles = cacheStyles((theme: Theme) => ({
     color: theme.primaryText,
     fontSize: theme.rem(1)
   },
+  bundles: {
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name